apps/notification_bot/utils.ts

```diff
@@ -1,8 +1,102 @@
-export const getMessage = (link, userName, users) => (`
-******************************
-Pull request opened.
-Author: ${userName}.
+import { Octokit } from '@octokit/core'
+const octokit = new Octokit({
+    auth: process.env.NOTIFICATION_BOT_CONFIG && JSON.parse(process.env.NOTIFICATION_BOT_CONFIG)?.github_auth_key,
+})
+const AVAILABLE_TASK_TYPES = ['SBERDOMA']
+
+export const getPullRequestMessage = (link, userName, users) => (`
+---------------------------------
+${userName} opened a pull request.
+${link}.
 ${users.map(([userName]) => `@${userName}`).join(', ')}
-Link: ${link}.
-******************************
-`)
\ No newline at end of file
+---------------------------------
+`)
+
+export const getFormattedCommits = (commits) => {
+    const formattedCommit = commits
+        .map(({ message }) => {
+            const [task_number] = message.split(' ')
+            return task_number
+        })
+        .filter((task_number) => {
+            const [taskType] = task_number.split('-')
+
+            return AVAILABLE_TASK_TYPES.includes(taskType)
+        })
```

Comment:
It seems, that more configurable approach is to use `reduce` instead of `map` with `TASK_NUMBER_REGEXP` to capture numbers instead of using `AVAILABLE_TASK_TYPES`. This way we will have one pass over array instead of two passes (with `map` and `filter`) and will not be tied to `split` and `-` char in algorithm.

Reply:
:)